modules/seqexec/server/src/main/scala/seqexec/server/SeqexecEngine.scala

```diff
@@ -847,10 +847,11 @@ object SeqexecEngine extends SeqexecConfiguration {
     case UpdateQueueClear(qid)              => QueueUpdated(QueueManipulationOp.Clear(qid), svs)
     case StartQueue(qid, _)                 => QueueUpdated(QueueManipulationOp.Started(qid), svs)
     case StopQueue(qid, _)                  => QueueUpdated(QueueManipulationOp.Stopped(qid), svs)
-    case StartSysConfig(_, _, _)            => NullEvent
+    case StartSysConfig(sid, stepId, res)   => SingleActionEvent(SingleActionOp.Started(sid, stepId, res))
   }
 
-  def toSeqexecEvent(ev: executeEngine.ResultType)(svs: => SequencesQueue[SequenceView]): SeqexecEvent = ev match {
+  def toSeqexecEvent(ev: executeEngine.ResultType, qState: EngineState)
+                    (svs: => SequencesQueue[SequenceView]): SeqexecEvent = ev match {
```

Comment:
`svs` could be removed and built from `qState`. The caller does that, and only builds it for calling `toSeqexcEvent`.

Reply:
Sure. I can do that, but some of the methods used are `private`, so I think I'll have to change that so I can access them. Whichever you prefer is fine with me.